internal/postgresql/postgresql.go

```diff
@@ -647,6 +651,41 @@ func (p *Manager) PGDataVersion() (int, int, error) {
 	return ParseVersion(version)
 }
 
+/* Both PostgreSQL 13.0beta1 and PostgreSQL 13 stable will have major version 13, minor 0.
```

Comment:
Since version 13 is our for a while now, isn't it better to only support 13 and not add extra code for beta1?